docs/root/api-docs/xds_protocol.rst

```diff
@@ -498,18 +498,20 @@ twice. Clients should NACK responses that contain multiple instances of the same
 Deleting Resources
 ^^^^^^^^^^^^^^^^^^
 
+Wildcard Resource Types
+^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
i think this needs to be a different type of header to `Deleting Resources` as these are subsections